gc/scheduler/scheduler_test.go

```diff
@@ -39,8 +40,9 @@ func TestPauseThreshold(t *testing.T) {
 	}()
 
 	time.Sleep(time.Millisecond * 15)
-	if c := tc.runCount(); c < 3 || c > 4 {
-		t.Fatalf("unexpected gc run count %d, expected between 5 and 6", c)
+
+	if c := tc.runCount(); c > 4 {
+		t.Fatalf("unexpected gc run count %d, expected less than 5", c)
```

Comment:
Have you ever seen this runCount be 0? Maybe there should still be a minimum of 1?